v3/soda/mango/mango.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "MF17",

```

Comment:
MF17? Or should it be renamed to Mango?

Reply:
I confirm that my product model is MF17, so MF17 is correct